regolith/helpers/u_institutionshelper.py

```diff
@@ -0,0 +1,188 @@
+"""
+Helper for updating/adding  to the projecta collection.
+"""
+from regolith.helpers.basehelper import DbHelperBase
+from regolith.fsclient import _id_key
+from regolith.tools import all_docs_from_collection, fragment_retrieval
+from itertools import chain
+import uuid
+import datetime as dt
+
+TARGET_COLL = "institutions"
+
+def subparser(subpi):
+    subpi.add_argument("institution_id",
+                       help="id of the institution, e.g., columbiau.")
+    subpi.add_argument("-n","--name",
+                        help="Full name of the institution")
+    subpi.add_argument("-i","--index",
+                       help="Index of the item in the enumerated list to update.",
+                       type=int)
+    subpi.add_argument("--city",
+                       help="The city where the institution is. "
+                            "Required for a new institution.")
+    subpi.add_argument("--country",
+                       help="The country where the institution is. "
+                            "Required for a new institution")
+    subpi.add_argument("-a", "--aka",
+                       nargs='+',
+                       help="List of all the different names this "
+                            "institution is known by.")
+    subpi.add_argument("--state",
+                       help="The state where the institution is.")
+    subpi.add_argument("--zip",
+                       help="zipcode of the institution.")
+    subpi.add_argument("--dept_id",
+                       help="e.g. physics.")
+    subpi.add_argument("--dept_name",
+                       help="Department canonical name, e.g., Department of Physics. "
+                            "Defaults to 'Department of {department id.}'")
```

Comment:
this is too much magic so please remove.  add "Required if --dept_id supplied and it is a new department"

Reply:
done!